mm/2s2h/Rando/Logic/Logic.h

```diff
@@ -138,6 +139,16 @@ inline std::string LogicString(std::string condition) {
     return condition;
 }
 
+inline uint8_t FoundOcarinaButtons() {
+    uint8_t foundButtons = 0;
+    for (int i = RANDO_INF_OBTAINED_OCARINA_BUTTON_A; i < RANDO_INF_OBTAINED_OCARINA_BUTTON_C_UP; i++) {
```

Comment:
This has to be inclusive. If I have two buttons and one of them is C Up, the scarecrow will not spawn.
```suggestion
    for (int i = RANDO_INF_OBTAINED_OCARINA_BUTTON_A; i <= RANDO_INF_OBTAINED_OCARINA_BUTTON_C_UP; i++) {
```